components/brave_wallet_ui/trezor/trezor.ts

```diff
@@ -0,0 +1,45 @@
+import TrezorConnect, { Success, Unsuccessful } from 'trezor-connect'
+import { HDNodeResponse } from 'trezor-connect/lib/typescript/trezor/protobuf'
+import {
+  TrezorCommand,
+  UnlockCommand,
+  UnlockResponse,
+  GetAccountsCommand,
+  TrezorGetAccountsResponse,
+  GetAccountsResponsePayload,
+  addTrezorCommandHandler
+} from '../common/trezor/trezor-messages'
+
+type TrezorGetPublicKeyResponse = Unsuccessful | Success<HDNodeResponse[]>
+const createUnlockResponse = (command: UnlockCommand, result: Boolean, error?: Unsuccessful): UnlockResponse => {
+  return { id: command.id, command: command.command, result: result, origin: command.origin, error: error }
+}
+
+const createGetAccountsResponse = (command: GetAccountsCommand, result: TrezorGetPublicKeyResponse): GetAccountsResponsePayload => {
+  return { id: command.id, command: command.command, payload: result as TrezorGetAccountsResponse, origin: command.origin }
```

Comment:
Just a question here - why cast between our type and the type from tezor library, why not import `TrezorGetPublicKeyResponse` in to trezor-message.ts just like you have already imported `Unsuccessful` type?

Reply:
done, thanks!